cmd/jag/commands/flash.go

```diff
@@ -101,11 +102,8 @@ func FlashCmd() *cobra.Command {
 			}
 			defer os.Remove(envelopeFile.Name())
 
-			if chip == "esp32s3-spiram-octo" {
-				chip = "esp32s3"
-			} else if chip == "esp32-eth-clk-out17" {
-				chip = "esp32"
-			}
+			// Split at first '-'.
```

Comment:
I assume this works even if there isn't a -?

Reply:
yes. Tested it locally.